edgelet/hsm-sys/azure-iot-hsm-c/src/edge_hsm_client_store.c

```diff
@@ -94,6 +94,19 @@ static HSM_STATE_T g_hsm_state = HSM_STATE_UNPROVISIONED;
 static CRYPTO_STORE* g_crypto_store = NULL;
 static int g_store_ref_count = 0;
 
+//##############################################################################
+// Helpers
+//##############################################################################
+
+#define FREEIF(x) \
```

Comment:
Move this to a common header instead of duplicating it between `certificate_info.c`, here and `edge_hsm_client_x509.c` ?

Reply:
Done.